table/partitioned_filter_block_test.cc

```diff
@@ -89,12 +89,10 @@ class PartitionedFilterBlockTest : public testing::Test {
 
   PartitionedFilterBlockBuilder* NewBuilder(
       PartitionedIndexBuilder* const p_index_builder) {
-    uint32_t partition_size =
-        table_options_.metadata_block_size >
-                (uint64_t)table_options_.block_size_deviation
-            ? table_options_.metadata_block_size -
-                  table_options_.block_size_deviation
-            : 1;
+    assert(table_opt.block_size_deviation <= 100);
+    auto partition_size = static_cast<const uint32_t>(
```

Comment:
auto is less favored here as it is not noisy nor obvious: https://google.github.io/styleguide/cppguide.html#auto

Similarly above.